wfl/fit/ace.py

```diff
@@ -132,14 +128,17 @@ def prepare_params(ACE_name, fitting_configs, ace_fit_params, run_dir='.', ref_p
 
     _prepare_e0(ace_fit_params, fitting_configs, ref_property_prefix)
 
-    if weight_from_sigma:
-        ace_fit_params["weights"] = {}
-        for at_i, at in enumerate(fitting_configs):
-            config_type = f"config_{at_i}"
-            at.info["config_type"] = config_type
-            ace_fit_params["weights"][config_type] = { "E": 1.0 / at.info.get("energy_sigma", 1.0),
-                                                       "F": 1.0 / at.info.get("force_sigma", 1.0),
-                                                       "V": 1.0 / at.info.get("virial_sigma", 1.0) }
+    from_sigma = ace_fit_params.get("weights", {}).get("from_sigma")
```

Comment:
Maybe would be good to mention this behaviour in docstring of this function or someplace else?

Reply:
Yes, good point. I'll update the docstring.

Reply:
addressed in bba5be1